pychromecast/__init__.py

```diff
@@ -349,20 +349,20 @@ def reboot(self):
         """ Reboots the Chromecast. """
         reboot(self.host)
 
-    def volume_up(self):
-        """ Increment volume by 0.1 unless it is already maxed.
+    def volume_up(self, delta=0.1):
+        """ Increment volume by 0.1 (or delta) unless it is already maxed.
         Returns the new volume.
 
         """
-        volume = round(self.status.volume_level, 1)
-        return self.set_volume(volume + 0.1)
+        volume = round(self.status.volume_level, 2)
```

Comment:
1. This PR is required (at least for me).  With a delta of `0.1`, it is not possible to select a reasonable volume level on one of my devices.  The "Volume Up" button on the device itself adjusts the volume by `0.02`, for example.

2. If you do rounding here, then you have to pick a number (`2` here).  Two significant digits seems a reasonable choice to me, three might be reasonable too. One digit is known to be inadequate, and four or more would be overkill.  So which number to pick?  `2` seems reasonable.

3. Why round at all?  It's not clear what the advantage of rounding is.  Why not just take it out and let the user decide.

4. There should probably be `Math.max(0.0, new_volume)` and `Math.min(1.0, new_volume)` in here somewhere too.